### PR TITLE
feat(antlers.html): add statamic antlers icon support

### PR DESCRIPTION
**Pull request by mytchallb**
_**Fixes #IssueNumber**_

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

The Statamic [branding](https://statamic.com/branding) icon to use for their unique *.antlers.html files.

### CODE REVIEW

**Comment by mytchallb** [comment]
Just added the icon file into the commit

File: icons/file_type_antlers_html.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....2577 1.0972-3.5449 2.1522-4.1568 0.31651-0.1899 0.31651-0.63301 0-0.84402-1.0972-0.69631-2.1522-2.1522-2.1522-4.0935v-4.6421c0-2.6164-0.71741-3.3761-3.0174-3.3761h-20.657c-2.2999 0-3.0174 0.75961-3.0174 3.3761v4.6632c0 1.9412-1.055 3.3972-2.1522 4.0935-0.31651 0.1899-0.31651 0.65411 0 0.84402 1.055 0.59081 2.1522 1.8779 2.1522 4.1568v4.7265c0 2.4265 0.84402 3.4183 3.144 3.4183zm-10.255-4.3256c-2.4687 0-4.2623-0.73851-5.9292-2.1944-0.2743-0.2532-0.37981-0.56971-0.37981-0.88622 0-0.25321 0.06331-0.52751 0.2532-0.73852l0.52751-0.67521c0.2532-0.3165 0.52751-0.46421 0.84402-0.46421 0.35871 0 0.69631 0.1477 1.055 0.37981 1.1394 0.73851 2.4054 1.1605 3.9247 1.1605 1.1605 0 2.0889-0.48531 2.0889-1.477 0-2.5742-8.5668-1.1605-8.5668-6.942 0-3.1018 2.532-4.832 5.8237-4.832 2.321 0 4.0302 0.67521 5.2962 1.477 0.31651 0.211 0.52751 0.63301 0.52751 1.055 0 0.2532-0.06331 0.48531-0.211 0.69631l-0.37981 0.56971c-0.27431 0.37981-0.59081 0.56971-0.99172 0.56971-0.27431 0-0.56971-0.1055-0.88622-0.2532-0.94952-0.48531-1.9412-0.73852-3.144-0.73852-1.2449 0-1.9834 0.67521-1.9834 1.3715 0 2.6376 8.5668 1.2027 8.5668 6.8154 0 3.1651-2.5531 5.1063-6.4356 5.1063z" fill="#01d7b0" stroke-width=".211"/></svg>
```
Comment (remcohaszing) [inline icons/file_type_antlers_html.svg:1]: The SVG could be made even smaller, but it already does fit within our 2KB file limit. It looks good enough to me, but I’ll leave the final review to @robertohuertasm.

**Review by remcohaszing — DISMISSED** [review]

File: icons/file_type_antlers_html.svg (hunk 1 above)
Comment (robertohuertasm) [reply icons/file_type_antlers_html.svg]: I've updated the icon according to the guidelines (2px border) and used inkscape recommended settings for optimization.

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

**Pull request merged**